lang/java/tools/src/main/java/org/apache/avro/tool/SpecificCompilerTool.java

```diff
@@ -73,6 +77,17 @@ public int run(InputStream in, PrintStream out, PrintStream err, List<String> ar
       arg++;
     }
 
+    if ("-fieldVisibility".equals(args.get(arg))) {
+      arg++;
+      try {
+        fieldVisibility = Optional.of(FieldVisibility.valueOf(args.get(arg).toUpperCase()));
```

Comment:
```suggestion
        fieldVisibility = Optional.of(FieldVisibility.valueOf(args.get(arg).toUpperCase(Locale.ENGLISH)));
```

Just to guard against the Turkish problem described in AVRO-2500!